### PR TITLE
Add the Redstone Controlled conveyor

### PR DESCRIPTION
**Pull request by BluSunrize**
See #3729

- It replaces the previous functionality of the normal conveyor to be disabled with redstone
- As a result, the "Redstone Ignoring" conveyor is now gone

ToDo: Despite what I wrote in the manual, we currently don't have an override for side-solidity in `IETileProviderBlock`, should probably add that.

### CODE REVIEW

File: src/main/java/blusunrize/immersiveengineering/common/blocks/metal/conveyors/RedstoneConveyor.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+/**
+ * @author BluSunrize - 06.05.2017
+ */
+public class RedstoneConveyor extends BasicConveyor
+{
+	public static final ResourceLocation NAME = new ResourceLocation(MODID, "redstone");
+
+	@OnlyIn(Dist.CLIENT)
+	private static final DynamicModel<Direction> MODEL_PANEL = DynamicModel.createSided(
```
Comment (malte0811) [inline src/main/java/blusunrize/immersiveengineering/common/blocks/metal/conveyors/RedstoneConveyor.java:53]: That will crash on servers, `OnlyIn` only removes the declaration, not the initialization.

File: src/main/resources/assets/immersiveengineering/manual/en_us/conveyors.txt [hunk 2]
```diff
@@ -3,8 +3,10 @@ Who needs rails anyway?
...
 Throwing items atop them will have the items move along the belt. Should the belt lead into an inventory, the item will be inserted.
 Blocks like hoppers will be able to drop items onto the belts (rightclick) or adjust whether they should run up- or downwards (sneak + rightclick).
-Using a redstone signal on a conveyor belt will turn it off, this behavior can be disabled by crafting it into a redstone-ignoring version.
 Rightclicking a conveyor with any dye will colour the walls of the conveyor to easily identify where is is going.<np>
+<&conveyor_redstone_recipe>The Redstone Controlled Conveyor Belt can be disabled with a redstone signal.
+The panel on its side allows attaching a lever, but a redstone signal may be input from any direction. Sneak + rightclick with an Engineer's Hammer to move the panel to the other side.<np>
+Using a redstone signal on a conveyor belt will turn it off, this behavior can be disabled by crafting it into a redstone-ignoring version.
```
Comment (malte0811) [inline src/main/resources/assets/immersiveengineering/manual/en_us/conveyors.txt:9]: Is that line still meant to be here?

File: src/main/java/blusunrize/immersiveengineering/common/blocks/metal/conveyors/RedstoneConveyor.java (hunk 1 above)
Comment (BluSunrize) [reply src/main/java/blusunrize/immersiveengineering/common/blocks/metal/conveyors/RedstoneConveyor.java:53]: Hmm, I think OnlyIn is not necessary here anyway, I just did it for brevity. I'll chuck it then.

File: src/main/resources/assets/immersiveengineering/manual/en_us/conveyors.txt (hunk 2 above)
Comment (BluSunrize) [reply src/main/resources/assets/immersiveengineering/manual/en_us/conveyors.txt:9]: Sure isn't.

**Comment by BluSunrize** [comment]
awaiting approval

**Pull request merged**